cgi-bin/LJ/Web.pm

```diff
@@ -886,7 +886,7 @@ sub create_qr_div {
     }
 
     return DW::Template->template_string( 'journal/quickreply.tt', {
-        form_url                => LJ::create_url( '/talkpost_do', host => $LJ::DOMAIN_WEB ),
```

Comment:
I don't get it. I don't think there is a use case for passing `$LJ::IS_SSL` as a value here.

I think that the `ssl` option should default to `$LJ::IS_SSL`. Then, anybody who _does not_ want to use that value would have to pass it in. Would that really break anything?